### PR TITLE
Create ui-c8y-1020-28-10-upgraded-angular-gettext-tools-to-fix-vulnerabilities-found-in-nth-check.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by pawel-rynarzewski-sag

Ticket: [MTM-58403](https://cumulocity.atlassian.net/browse/MTM-58403)
Original PR: [7214](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7214)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-28-10-upgraded-angular-gettext-tools-to-fix-vulnerabilities-found-in-nth-check.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Upgraded `angular-gettext-tools` to fix vulnerabilities found in `nth-check` and `cheerio` (#7214)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-28-10-upgraded-angular-gettext-tools-to-fix-vulnerabilities-found-in-nth-check.md]: The Claude AI suggested to use the following title:
```suggestion
title: Upgrade angular-gettext-tools to fix vulnerabilities
```

File: content/change-logs/application-enablement/ui-c8y-1020-28-10-upgraded-angular-gettext-tools-to-fix-vulnerabilities-found-in-nth-check.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58403
+version: 1020.28.10
+---
+Upgraded `angular-gettext-tools` to fix vulnerabilities found in `nth-check` and `cheerio` (#7214)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-28-10-upgraded-angular-gettext-tools-to-fix-vulnerabilities-found-in-nth-check.md]: The Claude AI suggested to use the following body:
```suggestion
The angular-gettext-tools library is used to extract translatable strings from source code in the Cumulocity UI. Security vulnerabilities were discovered in the nth-check and cheerio dependencies of angular-gettext-tools. To address these vulnerabilities, angular-gettext-tools has been upgraded to a newer version that includes fixed versions of the affected dependencies. This change is not expected to have any impact on the functionality of the Cumulocity UI.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@pawel-rynarzewski-sag Please check this PR.

**Review by pawel-rynarzewski-c8y — APPROVED** [review]

**Comment by pawel-rynarzewski-c8y** [comment]
> @pawel-rynarzewski-sag Please check this PR.

@BeateRixen, applied suggested improvements, looks good from technical perspective.

File: content/change-logs/application-enablement/ui-c8y-1020-28-10-upgraded-angular-gettext-tools-to-fix-vulnerabilities-found-in-nth-check.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58403
+version: 1020.28.10
+---
+The angular-gettext-tools library is used to extract translatable strings from source code in the Cumulocity UI. Security vulnerabilities were discovered in the nth-check and cheerio dependencies of angular-gettext-tools. To address these vulnerabilities, angular-gettext-tools has been upgraded to a newer version that includes fixed versions of the affected dependencies. This change is not expected to have any impact on the functionality of the Cumulocity UI.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-28-10-upgraded-angular-gettext-tools-to-fix-vulnerabilities-found-in-nth-check.md]:
```suggestion
The angular-gettext-tools library is used to extract translatable strings from the source code in the {{< product-c8y-iot >}} UI. Security vulnerabilities were discovered in the nth-check and cheerio dependencies of angular-gettext-tools. To address these vulnerabilities, angular-gettext-tools has been upgraded to a newer version that includes fixed versions of the affected dependencies. This change is not expected to have any impact on the functionality of the {{< product-c8y-iot >}} UI.
```

File: content/change-logs/application-enablement/ui-c8y-1020-28-10-upgraded-angular-gettext-tools-to-fix-vulnerabilities-found-in-nth-check.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Upgrade angular-gettext-tools to fix vulnerabilities
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-28-10-upgraded-angular-gettext-tools-to-fix-vulnerabilities-found-in-nth-check.md]:
```suggestion
title: Upgraded angular-gettext-tools library to fix vulnerabilities
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**